### PR TITLE
fix(modal container) modal container limitations

### PR DESCRIPTION
**Pull request by abirembaut**
- adapt modal container limitations to new behavior

### CODE REVIEW

File: md/widgets.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Each tab is a container widget.
 Use the **form container** to enable form validation for input widgets. The form container also exposes a local **$form** which is the AngularJS form object associated to the form container. In the Bonita Studio, if you define a contract for the task or process and click on the UI Designer icon, the generated form already contains a form container to embed default input widgets to collect the information that need to be passed to the contract to be verified. You can then define the validation you need on the input.  
 For example, you can bind the button's disabled property to _$form.$invalid_ to prevent user from clicking the button, until the form inputs are valid.
 
-### <a name="modal-widget"></a> Modal container widget
```
Comment (abirembaut) [inline md/widgets.md:56]: @NathalieC if you change the name, you need to change the link in this page : https://github.com/bonitasoft/bonita-doc/blob/7.8/md/uid-modal-tutorial.md

**Review by abirembaut — COMMENTED** [review]
The link is broken

File: md/widgets.md (hunk 1 above)
Comment (NathalieC) [reply md/widgets.md:56]: done.

Comment (NathalieC) [reply md/widgets.md:56]: PR#621

Comment (abirembaut) [reply md/widgets.md:56]: :+1: 
I think we should also change  "Tab container widget and "Form container widget" into "Tab container" and "Form container" in the previous sections

Comment (NathalieC) [reply md/widgets.md:56]: Yep. Done on 7.4 code. Here: PR#622

**Comment by abirembaut** [comment]
LGTM

**Pull request merged**